syntax = "proto3";

package fbsrankings.messages.query;

import "fbsrankings/messages/options/options.proto";

message SeasonByIDValue {
    string season_id = 1;
    int32 year = 2;
}

message SeasonByIDResult {
    string query_id = 1;
    optional SeasonByIDValue season = 2;
}

message SeasonByIDQuery {
    string query_id = 1;
    string season_id = 2;

    option (fbsrankings.messages.options.topic) = "fbsrankings.query.season_by_id";
}
